mllib/src/main/scala/org/apache/spark/ml/feature/PCA.scala

```diff
@@ -34,6 +34,7 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql._
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types.{StructField, StructType}
+import org.apache.spark.util.VersionUtils._
```

Comment:
I meant, are there others in the whole code base, but, I don't actually see any more. Can you address the import?